tests/integration/specs_test.py

```diff
@@ -459,21 +459,5 @@ def test_tlv_generating_tlv_error(tlv_message_value: pyrflx.MessageValue) -> Non
     assert tlv_message_value.bytestring == b"\x03"
 
 
-def test_tls(tmp_path: Path) -> None:
-    utils.assert_compilable_code_specs(
-        [
-            f"{EX_SPEC_DIR}/tls_alert.rflx",
-            f"{EX_SPEC_DIR}/tls_handshake.rflx",
-            f"{EX_SPEC_DIR}/tls_heartbeat.rflx",
-            f"{EX_SPEC_DIR}/tls_record.rflx",
-        ],
-        tmp_path,
-    )
-
-
-def test_icmp(tmp_path: Path) -> None:
-    utils.assert_compilable_code_specs([f"{EX_SPEC_DIR}/icmp.rflx"], tmp_path)
```

Comment:
Where did the ICMP and TLS integration tests go?

Reply:
They are done as part of the `RecordFlux-specifications` tests.  These tests are also executed in the CI of RecordFlux now (cf. 75890a1271b7e20ea50230fb4e410779559962dd).